src/include/daos_srv/pool.h

```diff
@@ -184,6 +187,11 @@ int ds_pool_cont_svc_lookup_leader(uuid_t pool_uuid, struct cont_svc **svc,
 
 void ds_pool_iv_ns_update(struct ds_pool *pool, unsigned int master_rank);
 
+int pool_iv_map_update(struct ds_pool *pool, struct pool_buf *buf,
+		       uint32_t map_ver);
+int pool_iv_prop_update(struct ds_pool *pool, daos_prop_t *prop);
+int pool_iv_prop_fetch(struct ds_pool *pool, daos_prop_t *prop);
```

Comment:
since you export those API to other server modules now, maybe should add ds_ prefix?

Reply:
ok